### ISSUE TITLE
Julia nightly and libcurl 8

### ISSUE THREAD

**Issue by joa-quim** [JuliaGeo/Proj.jl#80]
Hi, basically it's the same issue as this GDAL one https://github.com/JuliaGeo/GDAL.jl/issues/152
```
  | | |_| | | | (_| |  |  Version 1.10.0-DEV.1092 (2023-04-22)
 _/ |\__'_|_|_|\__'_|  |  Commit 357bcdd302 (2 days old master)
|__/                   |

(@v1.10) pkg> add PROJ_jll
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package LibCURL_jll [deac9b47]:
 LibCURL_jll [deac9b47] log:
 ├─possible versions are: 8.0.1 or uninstalled (package in sysimage!)
 └─found to have no compatible versions left with PROJ_jll [58948b4f]
```

**Comment by rafaqz** [comment]
PR would be nice ;)

**Comment by visr** [comment]
This should be fixed now with PROJ_jll v900.200.100.
